tools/Tools.Common/Loaders/MetadataLoader.cs

```diff
@@ -30,17 +30,21 @@ namespace Tools.Common.Loaders
 {
     public class MetadataLoader
     {
+        private static string _rootPath = Path.GetFullPath(Path.Combine(Assembly.GetExecutingAssembly().Location, "..", "..", ".."));
+
         public static ModuleMetadata GetModuleMetadata(string moduleName)
         {
-            string rootPath = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..", ".."));
-            string modulePsd1Path = Directory.GetFiles(Path.Combine(rootPath, "artifacts"), $"{moduleName}.psd1", SearchOption.AllDirectories)[0];
+            // bez: notice that this search way always find artifacts/Debug/{moduleName}/psd1 first, which may cause some issues
```

Comment:
Shall we need to add these two lines comments in our docs to maker scrum master to be aware of that?

Reply:
```
Build code
Clean the project.

<# Avoid issue of filename too long #>
git config --global core.longpaths true

<# Wipe any changes from a previous branch #>
git clean -xdf
```
Good suggestion. In our docs, we will clean all files under artifacts folder before build code. do you need me to add reasons here?